util.go

```diff
@@ -34,9 +34,6 @@ func IsFinalizationValid(eligibleSigners map[string]struct{}, finalization *Fina
 	if !valid {
 		return false
 	}
-	if !valid {
-		return false
-	}
 
 	return true
```

Comment:
we dont need the if at all, could just return below
```suggestion
	return valid
```